pkg/staticnetworkconfig/generator.go

```diff
@@ -248,20 +248,30 @@ func (s *StaticNetworkConfigGenerator) createNMConnectionFiles(nmstateOutput, ho
 	if len(connectionsList) == 0 {
 		return nil, errors.Errorf("nmstate generated an empty NetworkManager config file content")
 	}
+
+	var connectionFileNames []string
+
 	for _, connection := range connectionsList {
 		connectionElems := connection.([]interface{})
 		fileName := connectionElems[0].(string)
 		fileContents, err := s.formatNMConnection(connectionElems[1].(string))
 		if err != nil {
 			return nil, err
 		}
-		s.log.Infof("Adding NMConnection file <%s>", fileName)
+
 		newFile := StaticNetworkConfigData{
 			FilePath:     filepath.Join(hostDir, fileName),
 			FileContents: fileContents,
 		}
+
 		filesList = append(filesList, newFile)
+		connectionFileNames = append(connectionFileNames, fileName)
 	}
+
+	if len(connectionFileNames) > 0 {
+		s.log.Infof("Adding NMConnection files: <%s>", strings.Join(connectionFileNames, ">, <"))
+	}
```

Comment:
```suggestion
	if len(filesList) > 0 {
		s.log.Infof("Ading NMConnection files: <%s>",
			strings.Join(lo.Map(filesList, func(f StaticNetworkConfigData, _ int) string {
				return filepath.Base(f.FilePath)
			}), ">, <"),
		)
	}
```
What do you think?
I'm not sure which approach is better, but AFAIK, using append can be less efficient in terms of performance. I'm also not sure if it's worth creating a new list and using append just for an info log...

Reply:
I agree with you. I've changed that accordingly in the commit to improve performance after testing it passes the tests and works correctly in test-infra.